Makefile

```diff
@@ -287,6 +293,20 @@ test_stdlib: compile
 		cd lib/elixir && ../../bin/elixir --sname primary -r "test/elixir/test_helper.exs" -pr "test/elixir/**/$(TEST_FILES)"; \
 	fi
 
+cover/ex_unit_elixir.coverdata:
+	$(Q) COVER="1" $(MAKE) test_stdlib
```

Comment:
Is this a left-over?

Reply:
No, this is intentionally as a dependency of the `cover` target.

The idea is that you can call `make cover` and it will run all the tests with coverage. (I tried to make it simple for non-regular contributors as well to get the coverage report.)